Background job for mailer

eliseworthy opened the pull request:
I'm worried that this sends extra emails. I got emails when running tests. I have no idea why this would happen. Can someone test this locally?

Commit:
fixed signup workflow

athal7 commented:
getting a weird error here when trying to start the tests, did you do any other configuration locally? here's the error:

/Users/andrewthal/.rvm/rubies/ruby-1.9.3-p125/lib/ruby/1.9.1/uri/common.rb:176:in `split': bad URI(is not URI?):  (URI::InvalidURIError)
    from /Users/andrewthal/.rvm/rubies/ruby-1.9.3-p125/lib/ruby/1.9.1/uri/common.rb:211:in`parse'
    from /Users/andrewthal/.rvm/rubies/ruby-1.9.3-p125/lib/ruby/1.9.1/uri/common.rb:747:in `parse'
    from /Users/andrewthal/Dropbox/HungryAcademy/Code/feed_engine/config/initializers/redis.rb:1:in`<top (required)>'
    from /Users/andrewthal/.rvm/gems/ruby-1.9.3-p125@feed_engine/gems/activesupport-3.2.3/lib/active_support/dependencies.rb:245:in `load'
    from /Users/andrewthal/.rvm/gems/ruby-1.9.3-p125@feed_engine/gems/activesupport-3.2.3/lib/active_support/dependencies.rb:245:in`block in load'
    from /Users/andrewthal/.rvm/gems/ruby-1.9.3-p125@feed_engine/gems/activesupport-3.2.3/lib/active_support/dependencies.rb:236:in `load_dependency'
    from /Users/andrewthal/.rvm/gems/ruby-1.9.3-p125@feed_engine/gems/activesupport-3.2.3/lib/active_support/dependencies.rb:245:in`load'
    from /Users/andrewthal/.rvm/gems/ruby-1.9.3-p125@feed_engine/gems/railties-3.2.3/lib/rails/engine.rb:588:in `block (2 levels) in <class:Engine>'
    from /Users/andrewthal/.rvm/gems/ruby-1.9.3-p125@feed_engine/gems/railties-3.2.3/lib/rails/engine.rb:587:in`each'
    from /Users/andrewthal/.rvm/gems/ruby-1.9.3-p125@feed_engine/gems/railties-3.2.3/lib/rails/engine.rb:587:in `block in <class:Engine>'
    from /Users/andrewthal/.rvm/gems/ruby-1.9.3-p125@feed_engine/gems/railties-3.2.3/lib/rails/initializable.rb:30:in`instance_exec'
    from /Users/andrewthal/.rvm/gems/ruby-1.9.3-p125@feed_engine/gems/railties-3.2.3/lib/rails/initializable.rb:30:in `run'
    from /Users/andrewthal/.rvm/gems/ruby-1.9.3-p125@feed_engine/gems/railties-3.2.3/lib/rails/initializable.rb:55:in`block in run_initializers'
    from /Users/andrewthal/.rvm/gems/ruby-1.9.3-p125@feed_engine/gems/railties-3.2.3/lib/rails/initializable.rb:54:in `each'
    from /Users/andrewthal/.rvm/gems/ruby-1.9.3-p125@feed_engine/gems/railties-3.2.3/lib/rails/initializable.rb:54:in`run_initializers'
    from /Users/andrewthal/.rvm/gems/ruby-1.9.3-p125@feed_engine/gems/railties-3.2.3/lib/rails/application.rb:136:in `initialize!'
    from /Users/andrewthal/.rvm/gems/ruby-1.9.3-p125@feed_engine/gems/railties-3.2.3/lib/rails/railtie/configurable.rb:30:in`method_missing'
    from /Users/andrewthal/Dropbox/HungryAcademy/Code/feed_engine/config/environment.rb:5:in `<top (required)>'
    from /Users/andrewthal/Dropbox/HungryAcademy/Code/feed_engine/spec/spec_helper.rb:7:in`require'
    from /Users/andrewthal/Dropbox/HungryAcademy/Code/feed_engine/spec/spec_helper.rb:7:in `<top (required)>'
    from /Users/andrewthal/Dropbox/HungryAcademy/Code/feed_engine/spec/controllers/api/items_controller_spec.rb:1:in`require'
    from /Users/andrewthal/Dropbox/HungryAcademy/Code/feed_engine/spec/controllers/api/items_controller_spec.rb:1:in `<top (required)>'
    from /Users/andrewthal/.rvm/gems/ruby-1.9.3-p125@feed_engine/gems/rspec-core-2.10.1/lib/rspec/core/configuration.rb:746:in`load'
    from /Users/andrewthal/.rvm/gems/ruby-1.9.3-p125@feed_engine/gems/rspec-core-2.10.1/lib/rspec/core/configuration.rb:746:in `block in load_spec_files'
    from /Users/andrewthal/.rvm/gems/ruby-1.9.3-p125@feed_engine/gems/rspec-core-2.10.1/lib/rspec/core/configuration.rb:746:in`map'
    from /Users/andrewthal/.rvm/gems/ruby-1.9.3-p125@feed_engine/gems/rspec-core-2.10.1/lib/rspec/core/configuration.rb:746:in `load_spec_files'
    from /Users/andrewthal/.rvm/gems/ruby-1.9.3-p125@feed_engine/gems/rspec-core-2.10.1/lib/rspec/core/command_line.rb:22:in`run'
    from /Users/andrewthal/.rvm/gems/ruby-1.9.3-p125@feed_engine/gems/rspec-core-2.10.1/lib/rspec/core/runner.rb:69:in `run'
    from /Users/andrewthal/.rvm/gems/ruby-1.9.3-p125@feed_engine/gems/rspec-core-2.10.1/lib/rspec/core/runner.rb:10:in`block in autorun'

thejonanshow committed:
Level up API

The items#index action now recognizes 2 new parameters: 'all' and 'time'. If the
'all' parameter is supplied you will find all of the users posts under 'all' in
the items (e.g parsed_response['items']['all']). The 'time' parameter will
filter the users items by creation time given a UTC time string, returning all
items created on or after that time, including refeeds.

Commit:
Merge pull request #79 from eliseworthy/levelup_api

Level up API

eliseworthy commented:
Fixed!

eliseworthy commented:
Ughhh, there's a bug. Brb.

Commit:
Merge pull request #82 from eliseworthy/js-fix

Fix javascript workflow for services during signup

Commit:
Merge branch 'background-jobs' of github.com:eliseworthy/feed_engine into background-jobs

Commit:
add temple gem
